Source/ARTRest.m

```diff
@@ -298,6 +309,17 @@ - (NSString *)description {
         if (_options.clientId && !self.auth.isTokenAuth) {
             [mutableRequest setValue:encodeBase64(_options.clientId) forHTTPHeaderField:@"X-Ably-ClientId"];
         }
+        
+        if (_options.addRequestIds) {
+            if (fallbacks != nil) {
+                requestId = originalRequestId;
+            } else {
+                NSString *randomId = [NSUUID new].UUIDString;
+                requestId = [[randomId dataUsingEncoding:NSUTF8StringEncoding] base64EncodedStringWithOptions:0];
+            }
```

Comment:
it's not mine, I've just moved it from one place to another.
I guess its base64 because of this https://docs.ably.io/client-lib-development-guide/features/#RSC7c